modules/planning/open_space/trajectory_smoother/iterative_anchoring_smoother.cc

```diff
@@ -94,8 +94,10 @@ bool IterativeAnchoringSmoother::Smooth(
   DiscretizedPath interpolated_warm_start_path;
   // TODO(Jinyun): move to confs
   const double interpolated_delta_s = 0.1;
-  const double path_length = warm_start_path.Length();
-  for (double s = 0; s < path_length; s += interpolated_delta_s) {
+  double path_length = warm_start_path.Length();
+  double delta_s = path_length / std::ceil(path_length / interpolated_delta_s);
```

Comment:
@JasonZhou404 Need to protect against dividing by zero here.

Reply:
Not likely the two denominators would be zero, but is indeed a valid concern if the interpolated_delta_s by config is set as zero. Will fix it in future commits